test/performance/odp_l2fwd.c

```diff
@@ -1574,6 +1852,50 @@ static void create_groups(int num, odp_schedule_group_t *group)
 	}
 }
 
+static int set_vector_pool_params(odp_pool_param_t *params, odp_pool_capability_t pool_capa)
+{
+	uint32_t num_vec, vec_size;
+
+	if (gbl_args->appl.vec_size == 0)
+		vec_size = DEFAULT_VEC_SIZE;
+	else
+		vec_size = gbl_args->appl.vec_size;
+
+	ODPH_ASSERT(pool_capa.vector.max_size > 0);
+	if (vec_size > pool_capa.vector.max_size) {
+		if (gbl_args->appl.vec_size == 0) {
+			vec_size = pool_capa.vector.max_size;
+			printf("\nWarning: Vector size reduced to %u\n\n", vec_size);
+		} else {
+			ODPH_ERR("Error: Vector size too big %u. Maximum is %u.\n",
+				 vec_size, pool_capa.vector.max_size);
+			return -1;
+		}
+	}
+
+	if (gbl_args->appl.num_vec == 0)
+		num_vec = (params->pkt.num + vec_size - 1) / vec_size;
```

Comment:
`odp_pool_param_t *params` struct has been reinitialized by `odp_pool_param_init()` in `main()`, so the `params->pkt.num` value is not valid anymore. On linux-generic this causes `num_vec` value to be zero.

Reply:
Missed that one. Will use `-n, --num_pkt <num>` here.